taipy/core/_orchestrator/_dispatcher/_standalone_job_dispatcher.py

```diff
@@ -11,6 +11,7 @@
 
 from concurrent.futures import Executor, ProcessPoolExecutor
 from functools import partial
+from threading import RLock
```

Comment:
We are using a lock to access the shared variable `_nb_available_workers`

Reply:
I would revert this back to a normal Lock

Reply:
Yep I agree :D

Reply:
OK, then. I'll do the change.